CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - `apollo`
   - Fix configuration loading and schema tag support [#925](https://github.com/apollographql/apollo-tooling/pull/925)
+- `apollo-language-server`
+  - Fix language server mis-reporting client identity for schema loading operation [#940](https://github.com/apollographql/apollo-tooling/pull/940)
+- `vscode-apollo`
+  - Fix graphql comments not being highlighted correctly [#907](https://github.com/apollographql/apollo-tooling/pull/907)
```

Comment:
Thanks for adding this one in 😄

Reply:
![giphy-2](https://user-images.githubusercontent.com/9259509/51934207-10e39c80-23d1-11e9-90d0-be702a17dda5.gif)